packages/sakuli-legacy/src/context/sahi/action/common-action/common-actions.spec.ts

```diff
@@ -54,8 +59,32 @@ describe("common-actions", () => {
         await api._highlight(queryByLocator(By.css("#second")));
       });
 
+      it("should highlight via highlightElement", async () => {
+        // GIVEN
+        const highlightSpy = jest.spyOn(utilsModule, "highlightElement");
+        await driver.get(
+          mockHtml(`
+                        <ul>
+                            <li>First</li>
+                            <li id="second">Second</li>
+                            <li>Last</li>
+                        </ul>
+                    `)
+        );
+
+        // WHEN
+        await api._highlight(queryByLocator(By.css("#second")));
+
+        // THEN
+        expect(highlightSpy).toBeCalledTimes(1);
```

Comment:
One could verify the parameters passed to the highlight call. This would ensure the requested element is highlighted.